bitcoin/src/blockdata/transaction.rs

```diff
@@ -298,6 +310,9 @@ impl Sequence {
     /// disables relative lock time.
     pub const ENABLE_RBF_NO_LOCKTIME: Self = Sequence(0xFFFFFFFD);
 
+    /// The number of bytes that a sequence number contributes to the size of a transaction.
+    const SIZE: usize = 4; // Serialized length of a u32.
```

Comment:
If this is the serialized length of a u32, then why not just make this a u32 instead of a usize?

Reply:
Lengths are type `usize` in Rust by convention.